### PR TITLE
Improve uncommitted diff

### PR DESCRIPTION
**Pull request by guy-har**

### CODE REVIEW

File: pkg/graveler/uncommitted_diff_iterator.go [hunk 1]
```diff
@@ -109,5 +110,5 @@ func (d *uncommittedDiffIterator) Err() error {
 }
 
 func (d *uncommittedDiffIterator) Close() {
```
Comment (itaiad200) [inline pkg/graveler/uncommitted_diff_iterator.go:112]: Need to close the committedList as well.

**Review by itaiad200 — APPROVED** [review]
LGTM! Really simple and elegant solution. Approving but please fix the single comment (about the unclosed iterator)

**Review by itaiad200 — APPROVED** [review]
LGTM! Really simple and elegant solution. Approving but please fix the single comment (about the unclosed iterator)

File: pkg/graveler/uncommitted_diff_iterator.go (hunk 1 above)
Comment (guy-har) [reply pkg/graveler/uncommitted_diff_iterator.go:112]: Thanks

**Pull request merged**